w3af/plugins/audit/xss.py

```diff
@@ -158,6 +158,9 @@ def _identify_trivial_xss(self, mutant):
         ct_options, _ = response.get_headers().iget('X-Content-Type-Options')
         content_type, _ = response.get_headers().iget('Content-Type')
 
+        if content_type is not None:
+            content_type = content_type.split(';')[0].strip()
```

Comment:
I would add a comment above this line to explain why it was added.

Also, did you try to search through the rest of the plugins for `Content-Type` to see if similar issues exist?